src/Console/Command/WorkerCommand.php

```diff
@@ -0,0 +1,238 @@
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\ConsoleOutputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+/**
+ * @author Greg Korba <greg@codito.dev>
+ *
+ * @internal
+ */
+#[AsCommand(name: 'worker', description: 'Internal command for running fixers in parallel', hidden: true)]
+final class WorkerCommand extends Command
+{
+    protected static $defaultName = 'worker';
+    protected static $defaultDescription = 'Internal command for running fixers in parallel';
+
+    private ToolInfoInterface $toolInfo;
+    private ConfigurationResolver $configurationResolver;
+    private ErrorsManager $errorsManager;
+    private EventDispatcherInterface $eventDispatcher;
+
+    /** @var array<int, FixerFileProcessedEvent> */
+    private array $events;
+
+    public function __construct(ToolInfoInterface $toolInfo)
+    {
+        parent::__construct();
+
+        $this->setHidden(true);
+        $this->toolInfo = $toolInfo;
+        $this->errorsManager = new ErrorsManager();
+        $this->eventDispatcher = new EventDispatcher();
+    }
+
+    protected function configure(): void
+    {
+        $this->setDefinition(
+            [
+                new InputOption(
+                    'port',
+                    null,
+                    InputOption::VALUE_REQUIRED,
+                    'Specifies parallelisation server\'s port.'
+                ),
+                new InputOption(
+                    'identifier',
+                    null,
+                    InputOption::VALUE_REQUIRED,
+                    'Specifies parallelisation process\' identifier.'
+                ),
+                new InputOption(
+                    'allow-risky',
+                    '',
+                    InputOption::VALUE_REQUIRED,
+                    'Are risky fixers allowed (can be `yes` or `no`).'
+                ),
+                new InputOption('config', '', InputOption::VALUE_REQUIRED, 'The path to a config file.'),
+                new InputOption(
+                    'dry-run',
+                    '',
+                    InputOption::VALUE_NONE,
+                    'Only shows which files would have been modified.'
+                ),
+                new InputOption(
+                    'rules',
+                    '',
+                    InputOption::VALUE_REQUIRED,
+                    'List of rules that should be run against configured paths.'
+                ),
+                new InputOption(
+                    'using-cache',
+                    '',
+                    InputOption::VALUE_REQUIRED,
+                    'Does cache should be used (can be `yes` or `no`).'
```

Comment:
Wording can be improved I think:
```suggestion
                    'Should cache be used (can be `yes` or `no`).'
```
If updated here, the same change should be done in `FixCommand`.